### DISCUSSION TITLE
Could need some help to get anything to work / registration works but nothing else

### DISCUSSION THREAD

**Discussion by Syntoxr** [tayler6000/pyVoIP#161] [Q&A]
Hi,
I could need some help to get this to work. 
What confuses me at the moment is that when using soft phones like PhonerLite or Lin phone, everything seems to work fine (So I guess SIP is configured correctly on my router (fritzbox)). But when using this library with the given examples, I am able to log in to the sip server but nothing else seems to happen:

- when running I get to "Press enter to disable the phone" so it seems like I'm registered
- the script does not react to incoming calls, at least the callback does not seem to get triggered
- I can't call any number / calls are not going out
- (again) soft phone apps work like the should

Is there something else I have to do before I am able to use this library?

Any help is appreciated!
Thanks!

**Comment by wooshings** [comment]
did you find an answer? i'm having the same issue and the docs don't help.
